Cargo.toml

```diff
@@ -8,7 +8,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 # Default enable napi4 feature, see https://nodejs.org/api/n-api.html#node-api-version-matrix
```

Comment:
This comment is outdated now? I do not quite understand it (and it's purpose).

Reply:
some leftovers from napi auto-generated template